sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/PathsHelper.java

```diff
@@ -109,7 +111,7 @@ private static String generatePathForNameBased(ResourceType resourceType, String
         return resourcePath;
     }
 
-    public static String generatePath(ResourceType resourceType, String ownerOrResourceId, boolean isFeed) {
+    public static String generatePath(ResourceType resourceType, String ownerOrResourceId, boolean isFeed, OperationType operationType) {
```

Comment:
if you add an overload `PathHelper.generatePath` with operationType = null default. then you don't have to modify many other files.

Reply:
Done